src/store/translation/reducers.ts

```diff
@@ -0,0 +1,20 @@
+import { Language } from '../../util/types';
```

Comment:
Maybe this is better under UI reducers instead of creating a new folder for translations